app/Platform/Services/TicketViewService.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Services;
+
+use App\Community\Enums\TicketState;
+use App\Community\Enums\TicketType;
+use App\Enums\PlayerGameActivityEventType;
+use App\Models\PlayerAchievement;
+use App\Models\Ticket;
+use App\Models\User;
+
+class TicketViewService
+{
+    public int $unlocksSinceReported = 0;
+    public array $openTickets = [];
+    public array $closedTickets = [];
+    public string $contactReporterUrl = '';
+    public ?PlayerAchievement $existingUnlock = null;
+    public string $ticketNotes = '';
+
+    public function __construct(
+        public PlayerGameActivityService $activity = new PlayerGameActivityService(),
+        public UserAgentService $userAgentService = new UserAgentService(),
+    ) {
```

Comment:
```suggestion
    public function __construct(
        protected PlayerGameActivityService $activity,
        protected UserAgentService $userAgentService,
    ) {
```

Something external shouldn't be explicitly accessing `TicketViewService`'s instances of these two services. It should be okay to set them to `protected` and not need to `new` them up. This will cause Laravel DI to kick in.